sos/report/plugins/nvme.py

```diff
@@ -18,7 +18,11 @@ class Nvme(Plugin, IndependentPlugin):
     packages = ('nvme-cli',)
 
     def setup(self):
-        self.add_copy_spec("/etc/nvme/discovery.conf")
+        self.add_copy_spec([
+            "/etc/nvme/discovery.conf",
+            "/etc/nvme/hostnqn",
```

Comment:
At this point it may be better to capture the whole /etc/nvme directory. There's another file we can get: /etc/nvme/hostid , that can be useful to diagnose nvme issues, and users may have old discovery.conf files backed up there that can help understand changes made to the subsystem.

Reply:
Will update  patch for collecting /etc/nvme directory and send new version of patch